packages/server/modules/workspaces/services/management.ts

```diff
@@ -493,10 +495,10 @@ export const updateWorkspaceRoleFactory =
           workspaceId,
           role: nextWorkspaceRole
         },
+        updatedByUserId,
         flags: {
           skipProjectRoleUpdatesFor: skipProjectRoleUpdatesFor ?? []
```

Comment:
@cdriesler we can't get rid of skipProjectRoleUpdatesFor and the old role assignment logic yet, otherwise a bunch of tests that use old plans break. i'm handling that in a separate PR, so this PR should only fix logic for new plans without breaking logic for old ones